sdk/containerregistry/azure-containers-containerregistry/TROUBLESHOOTING.md

```diff
@@ -0,0 +1,127 @@
+# Troubleshoot Azure Container Registry client library issues
+
+This troubleshooting guide contains instructions to diagnose frequently encountered issues while using the Azure Container Registry client library for Java.
+
+## General Troubleshooting
+
+Container registry service methods throw a [HttpResponseException][HttpResponseException] on failure.
```

Comment:
This is a more general question, but does apply here, do we want to differentiate how exceptions are thrown in synchronous calls vs asynchronous calls? Ex, yes this will be thrown on a synchronous call but it will be returned through the error stream in asynchronous calls.